src/pages/landing/weekend-trading/components/_title-btn.js

```diff
@@ -1,49 +1,32 @@
 import React from 'react'
 import styled from 'styled-components'
+import PropTypes from 'prop-types'
 import device from 'themes/device'
 import { Container, SectionContainer } from 'components/containers'
 import { Text } from 'components/elements'
-import { localize } from 'components/localization'
 import { LinkButton } from 'components/form'
 
 const StyledSection = styled(SectionContainer)`
-    background-color: var(--color-grey-30);
-    margin-top: 67px;
-    padding: 7rem 0 3rem;
-
     @media ${device.tabletL} {
         padding: 40px 16px 0;
     }
 `
-const StyledContainer = styled(Container)`
-    flex-direction: column;
 
+const StyledContainer = styled(Container)`
     @media ${device.tabletL} {
         width: 100%;
     }
 `
 
 const StyledText = styled(Text)`
-    font-size: 3.2rem;
-    line-height: 40px;
-    text-align: center;
-    font-weight: 700;
-
     @media ${device.tabletL} {
         font-size: 24px;
         line-height: 30px;
+        text-align: center;
     }
 `
 
 const TryButton = styled(LinkButton)`
-    padding: 14px 16px;
-    width: min-content;
-    white-space: nowrap;
-    font-size: 14px;
-    margin-top: 4.2rem;
-    margin-bottom: 40px;
-    border: unset;
-
     @media ${device.mobileL} {
         padding: 1.5rem 1.6rem;
```

Comment:
This does not compute to 15px 16px as 1rem does not always equal to 10px here. It's more ideal to use pixels for this.

Reply:
updated